docs/_sidebar.md

```diff
@@ -1,4 +1,69 @@
 <!-- markdownlint-disable-next-line first-line-heading -->
 - [**Home**](/homepage.md)
-- [**User Guide**](userguide/intro.md)
-- [**API Reference**](api/ActivityMonitorPlugin.md)
\ No newline at end of file
+- [**User Guide**](/README)
```

Comment:
Can we just expand API Reference? We don't have to expand User Guide by default.

Reply:
Reverted the changes under UserGuide. Only expanded the sidebar under API reference